db/createOrUpdateDb.js

```diff
@@ -18,38 +18,19 @@ const models = require('./index');
 const pgtools = require('pgtools');
 const utils = require('./utils');
 
-/*
- * If this is running in a Heroku Private Space, just exit (ref.
- * createOrUpdatePrivateDb.js).
- */
-if (utils.isInHerokuPrivateSpace()) {
-  console.log('Exiting "./db/createOrUpdateDb" because db is running in a ' +
-    'Heroku Private Space');
-  process.exit(0); // eslint-disable-line
-}
-
-console.log('querying information_schema.tables where table_schema = \'public\'')
 models.sequelize.query(`select count(*) from
   information_schema.tables where table_schema = 'public'`)
 .then((data) => {
-  console.log('found', data)
+  console.log('found', data); // eslint-disable-line
```

Comment:
I added this when troubleshooting migration issues a while back. We don't need to keep this console.log anymore either, unless it's useful to you for migration troubleshooting